src/screens/Assets/components/ChainListHeader.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2021 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { BigNumber } from 'bignumber.js';
+import styled from 'styled-components/native';
+
+// Components
+import Text from 'components/modern/Text';
+
+// Selectors
+import { useFiatCurrency } from 'selectors';
+
+// Utils
+import { formatFiatValue } from 'utils/format';
+import { spacing } from 'utils/variables';
+import { useChainsConfig } from 'utils/uiConfig';
+
+// Types
+import { type Chain } from 'models/Chain';
+
+type Props = {|
+  chain: Chain,
+  onPress?: () => mixed,
+  balance?: ?BigNumber,
```

Comment:
As per other comment I think it's redundant to define prop type and it's value to `maybe`. Either prop is provided or not with right value or prop is not provided at all or prop is always provided, but value is `maybe`. Hope it makes sense 👀